pkg/controller/control.go

```diff
@@ -1062,14 +1065,16 @@ func (c *Controller) FrontendState() string {
 }
 
 func (c *Controller) recordMetrics(isRead bool, dataLength int, latency time.Duration) {
```

Comment:
Because `recordMetrics` is part of write and read method, will it impact the IO? If yes, probably we should record metrics asynchronously out from write/read like using a goruntine to do that. (metrics can be best efforts)

Reply:
> Because recordMetrics is part of write and read method, will it impact the IO? 

According to my experience, I think no. I'm running the benchmark, wll update it later.

Reply:
Don't see the performance difference between the two cases.

**Enable metrics**
```
        Random:          32,123 / 15,537
    Sequential:          29,562 / 27,833

Bandwidth in KiB/sec (Read/Write)
        Random:        347,891 / 413,658
    Sequential:        451,056 / 434,144

Latency in ns (Read/Write)
        Random:        458,593 / 393,689
    Sequential:        379,152 / 400,432
```

**Disable metrics**
```
IOPS (Read/Write)
        Random:          32,320 / 15,545
    Sequential:          29,348 / 27,903

Bandwidth in KiB/sec (Read/Write)
        Random:        348,873 / 414,618
    Sequential:        451,903 / 435,478

Latency in ns (Read/Write)
        Random:        461,520 / 397,417
    Sequential:        378,810 / 402,162
```

cc @shuo-wu @innobead